tests/qinfo/test_entropies.py

```diff
@@ -309,3 +315,310 @@ def circuit_state(x):
         grad_expected_entropy = expected_entropy_grad_ising_xx(param) / np.log(base)
 
         assert qml.math.allclose(grad_entropy, grad_expected_entropy, rtol=1e-04, atol=1e-05)
+
+
+class TestMutualInformation:
+    """Tests for the mutual information functions"""
+
+    @pytest.mark.parametrize("device", ["default.qubit", "default.mixed"])
```

Comment:
Let's parametrize over `lightning.qubit` too for each test case.

Reply:
Added `lightning.qubit` device to the (supported) tests.